ledger/test.go

```diff
@@ -0,0 +1,243 @@
+package ledger
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"fmt"
+	"math/rand"
+
+	"github.com/onflow/flow-go/ledger/common/hash"
+	"github.com/onflow/flow-go/ledger/common/utils"
+)
+
+// PathByUint8 returns a path (32 bytes) given a uint8
```

Comment:
if these are test related utilites I think it should be part of common/utils, the root of the ledger package is only kept for type definitions and errors.

Reply:
I moved them to the `ledger` to solve a circular dependency issue, but I'm not happy with this change either. I'll think of a better way.

Reply:
Done in https://github.com/onflow/flow-go/pull/514/commits/f8b83e880c7ea8eeceafb44afc1e1f24ea2dd1c7